statsmodels/sandbox/mice/mice.py

```diff
@@ -0,0 +1,1295 @@
+
+        Returns a MICEResults instance.
+        """
+
+        # Extract a few things from the models that were fit to
+        # imputed data sets.
+        params_list = []
+        cov_list = []
+        scale_list = []
+        for results in self.results_list:
+            params_list.append(np.asarray(results.params))
+            cov_list.append(np.asarray(results.cov_params()))
+            scale_list.append(results.scale)
+        params_list = np.asarray(params_list)
+        scale_list = np.asarray(scale_list)
+
+        # The estimated parameters for the MICE analysis
+        params = params_list.mean(0)
+
+        # The average of the within-imputation covariances
+        cov_within = sum(cov_list) / len(cov_list)
+
+        # The between-imputation covariance
+        cov_between = np.cov(params_list.T)
+
+        # The estimated covariance matrix for the MICE analysis
+        f = 1 + 1 / float(len(self.results_list))
+        cov_params = cov_within + f * cov_between
+
+        # Fraction of missing information
+        fmi = f * np.diag(cov_between) / np.diag(cov_params)
+
+        # Set up a results instance
+        scale = np.mean(scale_list)
+        results = MICEResults(self, params, cov_params / scale)
+        results.scale = scale
+        results.frac_miss_info = fmi
+        results.exog_names = self.exog_names
+        results.endog_names = self.endog_names
+        results.model_class = self.model_class
+
+        return results
+
+
+class MICEResults(LikelihoodModelResults):
+
+    def __init__(self, model, params, normalized_cov_params):
+
+        super(MICEResults, self).__init__(model, params,
+                                          normalized_cov_params)
+
+    def summary(self, title=None, alpha=.05):
+        """
+        Summarize the results of running MICE.
+
+        Parameters
+        -----------
+        title : string, optional
+            Title for the top table. If not None, then this replaces
+            the default title
+        alpha : float
+            Significance level for the confidence intervals
+
+        Returns
+        -------
+        smry : Summary instance
+            This holds the summary tables and text, which can be
+            printed or converted to various output formats.
+        """
+
+        from statsmodels.iolib import summary2
+        from statsmodels.compat.collections import OrderedDict
+
+        smry = summary2.Summary()
+        float_format = "%8.3f"
+
+        info = OrderedDict()
+        info["Method:"] = "MICE"
+        info["Model:"] = self.model_class.__name__
+        info["Dependent variable:"] = self.endog_names
+        info["Sample size:"] = "%d" % self.model.data.data.shape[0]
+        info["Scale"] = "%.2f" % self.scale
+        info["Num. imputations"] = "%d" % len(self.model.results_list)
+
+        smry.add_dict(info, align='l', float_format=float_format)
+
+        param = summary2.summary_params(self, alpha=alpha)
```

Comment:
Does this always use t-distribution for p-values and confidence interval?
I only see the table for the OLS examples in the notebooks.